tests/unit/.eslintrc.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  "extends": [
+    "plugin:vue/recommended",
+    "standard"
+  ],
+  "rules": {
+    "vue/max-attributes-per-line": [
+      "error",
+      {
+        "singleline": 5,
+        "multiline": {
+          "max": 1,
+          "allowFirstLine": false
+        }
+      }
+    ],
+    "semi": [
+      2,
+      "always",
+      {
+        "omitLastInOneLineBlock": true
+      }
+    ],
+    "no-var": 0,
+    "prefer-const": 1,
+    "no-undef": 1
+  },
+  "overrides": [
+    {
+      "files": [
+        "**/__tests__/*.{j,t}s?(x)"
+      ],
+      "env": {
```

Comment:
Is there a reason why this part of the configuration is under `overrides` and isn't just outside with everything else?

Reply:
It's copied from the lintrc file, so im not sure why it is this way but I didn't touch it